.gitignore

```diff
@@ -213,3 +213,8 @@ pip-log.txt
 
 #Mr Developer
 .mr.developer.cfg
+
+
+## Composer
+vendor
+composer.lock
```

Comment:
This doesn't really matter and pretty standard for stand alone libraries like this. If you are building project which depends upon certain version of certain library then its important to include  **composer.lock** file to enforce contributors to do `composer install` instead of `composer update` .

This library doesn't have any user-land dependencies and dev requirements are not that important for code execution.